src/receive_imf.rs

```diff
@@ -1693,12 +1693,6 @@ async fn add_parts(
             let name: &str = from.display_name.as_ref().unwrap_or(&from.addr);
             for part in &mut mime_parser.parts {
                 part.param.set(Param::OverrideSenderDisplayname, name);
```

Comment:
This usage of `OverrideSenderDisplayname` looks unnecessarily overloaded, i'd prefer we use smth else for this case. But maybe some translated string like "\~non-member: NAME" would be good, at least this "\~" won't cause confusion